data/strings/types_strings.txt

```diff
@@ -3191,6 +3093,7 @@
 
   [type.amenity.post_box]
     en = Post Box
+    en-GB = Postbox
```

Comment:
If this is where you mail letters, it is 'Mailbox".  However this refers to both the box you have where the Post Office puts your mail and the one that you put mail in.  They look different and the OSM object is about the one you put mail in.

Reply:
in the UK, incoming post boxes are called 'letterboxes' Changed en to Mailbox.